nixos/modules/services/monitoring/prometheus/exporters/nginx.nix

```diff
@@ -47,10 +47,10 @@ in
       ExecStart = ''
         ${pkgs.prometheus-nginx-exporter}/bin/nginx-prometheus-exporter \
           --nginx.scrape-uri '${cfg.scrapeUri}' \
-          --nginx.ssl-verify ${boolToString cfg.sslVerify} \
+          ${if cfg.sslVerify then "--nginx.ssl-verify" else ""} \
           --web.listen-address ${cfg.listenAddress}:${toString cfg.port} \
           --web.telemetry-path ${cfg.telemetryPath} \
-          --prometheus.const-labels ${concatStringsSep "," cfg.constLabels} \
+          ${if cfg.constLabels == [] then "" else "--prometheus.const-labels " (concatStringsSep "," cfg.constLabels)} \
```

Comment:
```suggestion
          ${lib.optionalString cfg.constLabels != [] "--prometheus.const-labels " (concatStringsSep "," cfg.constLabels)} \
```

Reply:
Both the change itself and the suggestion don't evaluate!

First of all, `"--prometheus.const-labels " (concatStringsSep ...` is missing a `+` (otherwise Nix interprets `--prometheus.const-labels` as function) and the condition after the `optionalString` is missing some brackets.